sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-data-inventory-report.sql

```diff
@@ -0,0 +1,507 @@
+                            where
+                                active_student_restrictions.student_id = students.id
+                                and active_student_restrictions.is_active <> true
+                        ) then ''Y''
+                        else ''N''
+                    end
+            ) as "Previous (inactive) Restrictions Flag",
+            (
+                select
+                    SUM (disbursement_overawards.overaward_value)
+                from
+                    sims.disbursement_overawards disbursement_overawards
+                where
+                    disbursement_overawards.student_id = students.id
+                    and disbursement_overawards.disbursement_value_code = ''CSLF''
+                GROUP by
+                    disbursement_overawards.student_id,
+                    disbursement_overawards.disbursement_value_code
+            ) as "CSL Overawards Balance (FT)",
+            (
+                select
+                    SUM (disbursement_overawards.overaward_value)
+                from
+                    sims.disbursement_overawards disbursement_overawards
+                where
+                    disbursement_overawards.student_id = students.id
+                    and disbursement_overawards.disbursement_value_code = ''CSLP''
+                GROUP by
+                    disbursement_overawards.student_id,
+                    disbursement_overawards.disbursement_value_code
+            ) as "CSL Overawards Balance (PT)",
+            (
+                select
+                    SUM (disbursement_overawards.overaward_value)
+                from
+                    sims.disbursement_overawards disbursement_overawards
+                where
+                    disbursement_overawards.student_id = students.id
+                    and disbursement_overawards.disbursement_value_code = ''BCSL''
+                GROUP by
+                    disbursement_overawards.student_id,
+                    disbursement_overawards.disbursement_value_code
+            ) as "BCSL Overawards Balance",
+            students.disability_status as "Permanent Disability status",
+            -- Application Personal Info (does not include Institution or Program info)
+            ''"'' || program_years.program_year_desc || ''"'' as "Program Year",
+            applications.data ->> ''studentNumber'' as "Student Number",
+            applications.data ->> ''citizenship'' as "Citizenship status",
+            applications.data ->> ''bcResident'' as "BC Residency Status",
+            applications.data ->> ''indigenousStatus'' as "Indigenous Person status",
+            applications.data ->> ''aboriginalHeritage'' as "Indigenous category code",
+            applications.data ->> ''applicationPDPPDStatus'' as "PD/PPD Application Flag",
+            applications.data ->> ''youthInCare'' as "Youth in Care Flag",
+            ''--TBD--'' as "Youth in Care beyond age 19 Flag",
```

Comment:
What is the question here? this is the youth in care at 19 right?

![image](https://github.com/bcgov/SIMS/assets/54600590/31bcd950-5664-4928-9beb-2f32cfe749f1)

Reply:
It seems so, but I'm not sure, so I left as TBD.